src/status_im/ui/screens/views.cljs

```diff
@@ -35,6 +37,19 @@
 
       [bottom-sheet/bottom-sheet opts])))
 
+(views/defview inactive-overlay []
+  (letsubs [app-state [:get :app-state]]
+    {:component-will-mount
+     (fn []
+       (react/dismiss-keyboard!))
+     :component-will-update
+     (fn []
+       (react/dismiss-keyboard!))}
```

Comment:
just wondering why do we need to dismiss keyboard on an update?

Reply:
oh :) I was looking for all the cases where some UI elements would render above the blanked out component and see if I could deal with to not make it so. Keyboard was the only one I could deal with like that :) in that hacky way. Couldn't deal with `alert` and `share` as you can see

should I omit it?

I should probably also not use :get subscription considering `re-frame subscriptions optimisation`